crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -17,9 +18,9 @@ use tracing::info;
 pub struct C8yEndPoint {
     pub c8y_host: String,
     pub device_id: String,
+    // internal id that can be used to build url, this could be main/child device internal-id
     pub c8y_internal_id: String,
```

Comment:
The `internal_id` here is for building the `http` request. This may be the `child or main` device internal ID based on the type of `http` request that is being built.